Learning_test_automation/Spotkanie6_Zadania/CurrencyConversion_vBeta/Option/Transaction.cs

```diff
@@ -7,20 +7,35 @@ public class Transaction
     {
         Option option = new Option();
         CurrencyPresentation currencyPresentation = new CurrencyPresentation();
-        Dollar dollar = new Dollar();
-        Euro euro = new Euro();
-        CzechCrown czechCrown = new CzechCrown();
+        Currency dollar = new Dollar();
+        Currency euro = new Euro();
+        Currency czechCrown = new CzechCrown();
+        List<Currency> currencies = new List<Currency>();
+
+        private decimal amount;
+
+        public Transaction()
+        {
+            currencies.Add(dollar);
+            currencies.Add(euro);
+            currencies.Add(czechCrown);
+        }
 
         public void Buying()
```

Comment:
Masz teraz kilka metod o nazwach zwiazanych z kupnem/sprzedaza, w stylu `Buying`, `Purchase`.
Propozycja to przeniesienie logiki zwiazanej z kupnem i sprzedaza do osobnych klas, dzieki temu bedzie kilka korzysci, zastanow sie jakie i pogadamy na mentoringu